src/box/lua/config/instance_config.lua

```diff
@@ -1118,6 +1118,67 @@ return schema.new('instance_config', schema.record({
             end
```

Comment:
> NO_DOC=in the subsequent patches in scope of #8861
> NO_CHANGELOG=in the subsequent patches in scope of #8861

Please, add a changelog entry.

Please, add more real reason to `NO_DOC`. Say,

> NO_DOC=https://github.com/tarantool/doc/issues/3544 links the most actual schema, no need to update the issue.

Reply:
> Please, add a changelog entry.

Ok. But I don't understand how it is helpful. I would say one changelog entry with "added all box.cfg options to config" would be more than enough.

Reply:
Changelogs with certain changes may be useful, when you're in a trouble and want to do some guesses about a reason of the trouble based on some reading, without real bisect effort. Sometimes it saves some time. It is a good thing to have, especially for a library or a developer platform.

Not a critical thing, but a nice thing and we all agreed to do that in tarantool.